src/core/xref.js

```diff
@@ -72,6 +87,50 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+/**
+ * converts conf.xref to object with url and spec properties
+ */
+function normalizeConfig(conf) {
+  const xrefDefaults = {
+    url: API_URL,
+    profile: "",
+    specs: null,
+  };
+
+  const normalizedConfig = {};
+  const type = Array.isArray(conf.xref) ? "array" : typeof conf.xref;
+  switch (type) {
+    case "boolean":
+      // ... true
+      Object.assign(normalizedConfig, xrefDefaults);
+      break;
+    case "string":
+      if (profiles[conf.xref]) {
+        // We have the profile
+        Object.assign(normalizedConfig, xrefDefaults, {
+          specs: profiles[conf.xref],
+        });
+      } else {
+        pub("error", "Invalid cross-references profile");
```

Comment:
Make sure you make this more descriptive. It should tell the user what profile is wrong, what the right options are, and how to fix the problem.

Reply:
I agree that I need to make this more descriptive, but how do I do that? The list of available profiles will expand when this feature is implemented, so is this a better message:
`Invalid cross-references profile {nameOfProfile}. Please use one of the supported profile configurations listed <a href=linkToWiki>here</a>.

Perhaps, we could maintain a list of supported profiles on the Wiki?